inst/extdata/energy/A15.roofPV_TechChange.csv

```diff
@@ -0,0 +1,5 @@
+# File: A15.roofPV_TechChange.csv,,,,,,,
+# Title: GCAM 3.0 rooftop PV techChange,,,,,,,
+# Units: Unitless,,,,,,,
+resource,subresource,2010,2025,2040,2055,2070,2085
```

Comment:
Don't know the exact source. Looks to have been generated internally. I did include a comment of where the initial values might have came from.